.circleci/build.sh

```diff
@@ -57,17 +57,33 @@ cd ${GLOW_DIR}
 mkdir build && cd build
 CMAKE_ARGS=("-DCMAKE_CXX_FLAGS=-Werror")
 CMAKE_ARGS+=("-DGLOW_WITH_CPU=ON")
-if [[ "$CIRCLE_JOB" == ASAN ]]; then
-    CMAKE_ARGS+=("-DGLOW_USE_SANITIZER='Address;Undefined'")
-fi
-if [[ "$CIRCLE_JOB" == DEBUG ]]; then
-    install_pocl
-    CMAKE_ARGS+=("-DCMAKE_BUILD_TYPE=Debug")
-    CMAKE_ARGS+=("-DGLOW_WITH_OPENCL=ON")
-else
-    CMAKE_ARGS+=("-DGLOW_WITH_OPENCL=OFF")
-    CMAKE_ARGS+=("-DCMAKE_BUILD_TYPE=Release")
-fi
+
+case "${CIRCLE_JOB}" in
+    ASAN)
+        CMAKE_ARGS+=("-DGLOW_USE_SANITIZER='Address;Undefined'")
+        CMAKE_ARGS+=("-DGLOW_WITH_OPENCL=OFF")
+        CMAKE_ARGS+=("-DCMAKE_BUILD_TYPE=Release")
+        ;;
+
+    DEBUG)
+        install_pocl
+        CMAKE_ARGS+=("-DCMAKE_BUILD_TYPE=Debug")
+        CMAKE_ARGS+=("-DGLOW_WITH_OPENCL=ON")
```

Comment:
for brevity we could add to default options right after line 59 and only turn off for ASAN

Reply:
done!